src/Components/SuperAdmins/super-admins.module.css

```diff
@@ -103,21 +138,23 @@
   background-color: transparent;
   background-image: url(../../../public/assets/images/super-admin-edit-icon.png);
   background-size: cover;
-  cursor:pointer;
+  cursor: pointer;
 }
 
 .passBtn {
```

Comment:
Same here

Reply:
They were left over from the original css and I forgot to remove them. They are currently not used.